nixos/modules/services/web-servers/caddy.nix

```diff
@@ -21,7 +21,7 @@ let
   '';
   tlsJSON = pkgs.writeText "tls.json" (builtins.toJSON tlsConfig);
   configJSON = pkgs.runCommand "caddy-config.json" { } ''
-    ${pkgs.jq}/bin/jq -s '.[0] * .[1]' ${adaptedConfig} ${tlsJSON} > $out
+    ${pkgs.jq}/bin/jq -s '.[0] * {"apps": {"tls": {"automation": {"policies": (if .[0].apps.tls.automation.policies == .[1]?.apps.tls.automation.policies then .[0].apps.tls.automation.policies else (.[0].apps.tls.automation.policies + .[1]?.apps.tls.automation.policies) end)}}}}' ${adaptedConfig} ${tlsJSON} > $out
```

Comment:
can we put this in a let expression with a comment? (that being the complicated part of the eq expression

Reply:
Done and reformatted much better :)